src/main/scala/com/monsanto/arch/cloudformation/model/resource/S3.scala

```diff
@@ -119,9 +119,14 @@ object S3LoggingConfiguration extends DefaultJsonProtocol {
   implicit val format: JsonFormat[S3LoggingConfiguration] = jsonFormat2(S3LoggingConfiguration.apply)
 }
 
-case class TopicConfigurations(TopicConfigurations: Seq[TopicConfiguration])
-object TopicConfigurations extends DefaultJsonProtocol {
-  implicit val format: JsonFormat[TopicConfigurations] = jsonFormat1(TopicConfigurations.apply)
+case class NotificationConfiguration(TopicConfigurations: Option[Seq[TopicConfiguration]] = None, LambdaConfigurations : Option[Seq[LambdaConfiguration]] = None)
```

Comment:
Can we add QueueConfigurations  too https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket-notificationconfig-queueconfig.html